pkgs/tools/security/noseyparker/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, cmake
+, openssl
+, pkg-config
+, hyperscan
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "noseyparker";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "praetorian-inc";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21